src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -252,13 +253,28 @@ export class XtermTerminal extends DisposableStore implements IXtermTerminal {
 	}
 
 	async findNext(term: string, searchOptions: ISearchOptions): Promise<boolean> {
+		this._updateFindColors(searchOptions);
 		return (await this._getSearchAddon()).findNext(term, searchOptions);
 	}
 
 	async findPrevious(term: string, searchOptions: ISearchOptions): Promise<boolean> {
+		this._updateFindColors(searchOptions);
 		return (await this._getSearchAddon()).findPrevious(term, searchOptions);
 	}
 
+	private _updateFindColors(searchOptions: ISearchOptions): void {
+		let selectedColor = this._configurationService.getValue('workbench.colorCustomizations.editor.findMatchBackground');
```

Comment:
aligns with what the editor does - can create an issue if we want to allow setting these for the terminal